src/main/java/org/craftercms/studio/impl/v2/service/dependency/DependencyServiceImpl.java

```diff
@@ -71,4 +76,12 @@ public List<String> getDependentItems(@ProtectedResourceId(SITE_ID_RESOURCE_ID)
         siteService.checkSiteExists(siteId);
         return dependencyServiceInternal.getDependentItems(siteId, paths);
     }
+
+    @Override
+    @HasPermission(type = DefaultPermission.class, action = PERMISSION_CONTENT_READ)
```

Comment:
Don't we need to annotate the siteId and paths params with `@ProtectedResourceId`?

Reply:
We do need the `@ProtectedResourceId` here for `siteId` and `path` parameters. I fixed this. Thank you!